notebooks/ingestion/Loading_CSVParsers.ipynb

```diff
@@ -4,49 +4,69 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Loading the data into the DB, using CSV Parsers (2/3).\n",
+    "# Ingest Cookbook (CSVParser).\n",
     "\n",
-    "Following the concepts on the different [means](/HowToGuides/start/Ingestion) of ingesting the data, we will build an example using the CSV Parser method in this notebook.\n",
+    "Following the concepts on the different [means](https://docs.aperturedata.io/HowToGuides/Ingestion/Ingestion#ingestion-using-data-from-csv-files) of ingesting the data, we will build an example using the CSV Parser method in this notebook.\n",
```

Comment:
can't we keep this as a relative url? like this won't it pull you from the dev docs site to the main?

Reply:
I had the same dilemma, because it should co locate with the docs environment. The reason I made it so, is for someone who opens it as a NB, has a brief intro with working references.

Reply:
gotcha,